src/CourierClient.php

```diff
@@ -9,12 +9,46 @@
 use Psr\Http\Message\ResponseInterface;
 use Shuttle\Shuttle;
 
-final class CourierClient
+interface CourierClientInterface {
+    public function getAuthorizationHeader(): string;
```

Comment:
This seems like it might be more suited to being a protected member, since it seems to be used only internally to the class. Thoughts on that?

Reply:
I think it makes sense